.github/workflows/release_notes_comments_migration.yml

```diff
@@ -0,0 +1,94 @@
+name: Copy Release Notes to Related Issues
+
+on:
+  pull_request:
+    types: [closed]
+    branches: [ master ]
+
+jobs:
+  copy_release_notes:
+    if: github.event.pull_request.merged == true
+    runs-on: ubuntu-latest
+    steps:
+      - name: Fetch PR Comments
+        id: get-comments
+        uses: actions/github-script@v7
+        with:
+          github-token: ${{secrets.GITHUB_TOKEN}}
+          script: |
+            const prNumber = context.payload.pull_request.number;
+            const repoName = context.repo.repo;
+            const repoOwner = context.repo.owner;
+            const releaseNotesRegex = /release notes/i;
+            
+            const comments = await github.rest.issues.listComments({
+              owner: repoOwner,
+              repo: repoName,
+              issue_number: prNumber,
+            });
+            
+            const releaseNoteComment = comments.data.find(comment => releaseNotesRegex.test(comment.body));
+            const releaseNoteBody = releaseNoteComment ? releaseNoteComment.body : '';
+            console.log(`Release Note Body: ${releaseNoteBody}`);
+            core.setOutput('releaseNoteBody', releaseNoteBody);
+
+      - name: Print Extracted releaseNoteBody
+        run: |
+          echo "Extracted Release Note Body:"
+          echo "${{ steps.get-comments.outputs.releaseNoteBody }}"
+          echo "RELEASE_NOTE_BODY<<EOF" >> $GITHUB_ENV
+          echo "${{ steps.get-comments.outputs.releaseNoteBody }}" >> $GITHUB_ENV
+          echo "EOF" >> $GITHUB_ENV
+
+      - name: Parse PR Description for Related Issues
+        id: find-issues
+        uses: actions/github-script@v7
+        with:
+          github-token: ${{secrets.GITHUB_TOKEN}}
+          script: |
+            const description = context.payload.pull_request.body;
+            const issueNumbers = [];
+            const regexPattern = /([Cc]los(e|es|ed)|[Ff]ix(es|ed)?|[Rr]esolv(e|es|ed))\s*#\s*([0-9]+)/g;
+            
+            let match;
+            while ((match = regexPattern.exec(description)) !== null) {
```

Comment:
I have an issue to wrap my head around this one

Reply:
This is a regex detector of the mentioned Issue in the PR description. It does not looks nice, yes.